jquery.balancetext.js

```diff
@@ -282,7 +282,11 @@
     $.fn.balanceText = function (skipResize) {
         var selector = this.selector;
 
-        if (!skipResize && balancedElements.indexOf(selector) === -1) {
+        if (!selector && this[0].className) {
+            selector = "." + this[0].className;
+        }
+
+        if (!skipResize && !!selector && balancedElements.indexOf(selector) === -1) {
```

Comment:
The other option I see is to change `balanceText()` function to take a selector string (instead of a `skipResize` boolean). Passing a selector implies that you want to balance on resize, which reverses the default when not passing a parameter, so this is an API breaking change.

Maybe take this change for now, and then change the API in v2.0.0 (along with some other PRs that I still need to review).